### PR TITLE
fix(web): delete trashed item

### PR DESCRIPTION
**Pull request by martabal**
fix for #8507, do not try to trash items which are already trashed

### CODE REVIEW

File: web/src/lib/components/photos-page/asset-grid.svelte [hunk 1]
```diff
@@ -71,7 +71,17 @@
...
+
+    if (hasTrashedAsset) {
+      if ($showDeleteModal) {
+        isShowDeleteConfirmation = true;
+        return;
+      } else {
+        handlePromiseError(trashOrDelete(true));
+        return;
+      }
+    } else if (!isTrashEnabled && $showDeleteModal) {
```
Comment (danieldietzler) [inline web/src/lib/components/photos-page/asset-grid.svelte:84]: When you're in the else case `isTrashEnabled` is already false, so the first part of the and is tautological.

**Review by danieldietzler — APPROVED** [review]

File: web/src/lib/components/photos-page/asset-grid.svelte (hunk 1 above)
Comment (martabal) [reply web/src/lib/components/photos-page/asset-grid.svelte]: Why is it already false ?

Comment (danieldietzler) [reply web/src/lib/components/photos-page/asset-grid.svelte]: Oh nvm, I misread `hasTrashedAsset` in l. 76 for `isTrashEnabled`. Sorry!

Comment (danieldietzler) [reply web/src/lib/components/photos-page/asset-grid.svelte]: But your new commit still looks better haha

Comment (martabal) [reply web/src/lib/components/photos-page/asset-grid.svelte]: > Oh nvm, I misread `hasTrashedAsset` in l. 76 for `isTrashEnabled`. Sorry!

np

> But your new commit still looks better haha

yeah, my first commit was a quick and ugly fix :sweat_smile:

**Review by jrasm91 — APPROVED** [review]

**Pull request merged**